src/Farmer/Arm/Web.fs

```diff
@@ -246,3 +249,33 @@ type StaticSite =
             this.RepositoryToken
         ]
 
+[<AutoOpen>]
+module SiteExtensions =
+    type SiteExtension =
+        { SiteName  : ResourceName
+          Name      : ResourceName
+          Location  : Location
+          }
+        interface IArmResource with
+            member this.ResourceName = this.SiteName / this.Name
+
+            member this.JsonModel =
+                siteExtensions.Create(
+                    this.Name,
+                    this.Location,
+                    [ ResourceId.create( sites, this.SiteName) ] ) :> _
+
+    //
```

Comment:
Can remove commented out code.

Reply:
Well, it was more to show what we were aiming for in the final template ; documentational, not code. I'll take it out though and I won't miss it. :-)